Rocket.Chat/Controllers/Chat/ChatControllerMessageCellProtocol.swift

```diff
@@ -19,6 +19,34 @@ extension ChatViewController: ChatMessageCellProtocol {
         }
     }
 
+    func handleUsernameTapMessageCell(_ message: Message, view: UIView, recognizer: UIGestureRecognizer) {
+        guard let username = message.user?.name else { return }
```

Comment:
If you need the user's username, you need to use the property `username` instead of `name`.